lib/product.rb

```diff
@@ -2,19 +2,16 @@ class Product
   attr_reader :category,
               :name,
               :unit_price,
-              :quantity,
-              :is_hoarded
```

Comment:
Since `is_hoarded` is referring to a method, there is no need to create an `attr_reader`. When an instance variable is added as an `attr_reader` it basically makes it so it works like a method. For instance, `product.quantity` would also work if instead of `:quantity` as an `attr_reader` you created a method called `quantity` that pointed back to `@quantity`.